scipy/integrate/_quadpack_py.py

```diff
@@ -1062,11 +1065,12 @@ def nquad(func, ranges, args=None, opts=None, full_output=False):
     The following provides a short description from [1]_ for each
     routine.
 
-    qagse
+    qagse (QAGS)
```

Comment:
I think this might just add confusion, I'd prefer to leave this part
```suggestion
    qagse
```